blocks/library/embed/index.js

```diff
@@ -1,7 +1,9 @@
 /**
  * WordPress dependencies
  */
-import { Button, Placeholder } from 'components';
+import Button from 'components/button';
+import Placeholder from 'components/placeholder';
+import HtmlEmbed from 'components/html-embed';
```

Comment:
I think if you rebase, you'll have to use `import { Button, Placeholder, HtmlEmbed } from 'components';`. We're using this new syntax to avoid duplicating the code across built files